api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -1272,7 +1272,9 @@ public static void stopModule(Module mod, boolean isShuttingDown) {
 	}
 
 	/**
-     * Merged collapsible if statement
+	 * Replaces further nested if statement with && on the outer if statement instead
```

Comment:
Replaces... etc is not helpful because the method consumer will not care whether this was a refactoring or not. Just describe what the method actually does. All that one wants is what the method will accomplish for them.